.gitignore

```diff
@@ -12,3 +12,9 @@ matrix.log
 glitch_plan*.yaml
 .tox
 .cache
+# exclude venv in case of "virtualenv ."
+lib/
+bin/
+local/
+include/
+pip-selfcheck.json
```

Comment:
I think that, since these should never exist, we shouldn't add them to the gitignore.

I'd like to be able to run "git status", say "whoa, what happened?", and then use the status output to figure out what needs to be removed.

We can prevent those dirs from getting checked in via the usual review method.